### ISSUE TITLE
Improve README to avoid questions due to connection issues

### ISSUE THREAD

**Issue by johanneskastl** [ygalblum/quadlet-demo#5]
TL;DR: I would prepare a PR to make it more clear in the README, where to connect to to get the Wordpress page (port 8000 AND HTTPS... :-) )

## For the next one stumbling...

I used the current state in the repository to try this out on a fresh CentOS 9 Stream VM (created by vagrant-libvirt, see [this repository](https://github.com/johanneskastl/quadlet-demo_vagrant_libvirt). But I get no connection to the wordpress instance.

```bash
# curl localhost:8000
curl: (52) Empty reply from server
#
```

The playbook runs successfully. The services are being generated and seem to be running:

```bash
# systemctl list-units|grep quadlet
  quadlet-demo-mysql-volume.service loaded active exited    quadlet-demo-mysql-volume.service
  quadlet-demo-mysql.service                loaded active running   quadlet-demo-mysql.service
  quadlet-demo-network.service            loaded active exited    quadlet-demo-network.service
  quadlet-demo.service                            loaded active running   quadlet-demo.service
#
```
I disabled firewalld and rebooted the machine, but to no avail.

I re-enabled firewalld, allowed the ports 8000, 8080 and 9000. I can get the envoy page at port 9000, but nothing on port 8000:

```bash
[root@quadlet-demo ~]# podman ps -a
CONTAINER ID  IMAGE                                    COMMAND               CREATED         STATUS         PORTS                                           NAMES
216090da2d12  docker.io/library/mysql:5.6              mysqld                19 minutes ago  Up 19 minutes                                                  quadlet-demo-mysql
fb4b6dfe5b94  localhost/podman-pause:4.4.0-1675439377                        19 minutes ago  Up 19 minutes                                                  6d2100dbe50d-service
24f5e5b2c116  localhost/podman-pause:4.4.0-1675439377                        19 minutes ago  Up 19 minutes  0.0.0.0:8000->8080/tcp, 0.0.0.0:9000->9901/tcp  ce83470d3dbf-infra
14d6f2215c99  docker.io/library/wordpress:4.8-apache   apache2-foregroun...  19 minutes ago  Up 19 minutes  0.0.0.0:8000->8080/tcp, 0.0.0.0:9000->9901/tcp  quadlet-demo-wordpress
ae38b041c06d  docker.io/envoyproxy/envoy:v1.25.0       envoy -c /etc/env...  19 minutes ago  Up 19 minutes  0.0.0.0:8000->8080/tcp, 0.0.0.0:9000->9901/tcp  quadlet-demo-envoy
[root@quadlet-demo ~]#
```
Finally I tried to hit this directly with HTTPS, got a certificate warning and the finally the wordpress page. Hooray!
